### ISSUE TITLE
Add contributors.md

### ISSUE THREAD

**Issue by annawoodard** [Parsl/parsl#393]

**Comment by benclifford** [comment]
`git log --pretty="%an%n%cn" | sort | uniq` will give something to start from

**Comment by kylechard** [comment]
After discussion we think this is unnecessary. It is easy to see a list of contributors via GitHub and we now have the codemeta file.

**Comment by danielskatz** [comment]
I disagree - contributors are not just the people who made commits in GH.
And CodeMeta is not easy to read.
